engine/resources/exec.go

```diff
@@ -800,3 +821,21 @@ func (obj *wrapWriter) Write(p []byte) (int, error) {
 func (obj *wrapWriter) String() string {
 	return obj.Buffer.String()
 }
+
+// checks that environment variable name is valid
```

Comment:
docstring in wrong format for two reasons. Please fix and tell me what the two mistakes are.

Reply:
No trailing period, and the comment doesn't begin with the function name.

Reply:
Bingo =D